docs/src/performance.md

```diff
@@ -267,3 +267,13 @@ requires. It can thus be seen as a proxy for "energy used" and, as an extension,
     timing result, you need to set the analysis interval such that the
     `AnalysisCallback` is invoked at least once during the course of the simulation and
     discard the first PID value.
+
+## Performance issues due to false sharing
```

Comment:
If people already know about false sharing, they don't need this section. Thus, I suggest to rename it to describe the problematic behavior, e.g.,

```suggestion
## Performance issues with multi-threaded reductions
```